Signal/src/view controllers/CallViewController.swift

```diff
@@ -187,15 +81,15 @@ class CallViewController: UIViewController, CallDelegate {
         contactsManager = Environment.getCurrent().contactsManager
         let callService = Environment.getCurrent().callService!
         callUIAdapter = callService.callUIAdapter
-        audioService = CallAudioService()
+        audioService = CallAudioService(handleRinging: callUIAdapter.hasManualRinger)
```

Comment:
That's valid. I'm not really sure where it belongs - it's behavior is different depending on CallKit vs. Not.

Maybe it belongs on the callUIAdapter.